components/Main/Result/Item.vue

```diff
@@ -98,6 +98,9 @@
 
         methods: {
             formatQuestionName(text) {
+                if (_startCase(text) === 'Serivce') {
```

Comment:
Như này thôi chứ nhỉ?
```suggestion
                if (text === 'Serivce') {
```